crate_universe/src/utils/starlark/select.rs

```diff
@@ -77,6 +77,19 @@ impl<T: Ord> SelectList<T> {
     pub fn is_empty(&self) -> bool {
         self.common.is_empty() && self.selects.is_empty() && self.unmapped.is_empty()
     }
+
+    /// Maps configuration names by `f`. This function must be injective
+    /// (that is a != b --> f(a) != f(b)).
```

Comment:
Small nit, this will render nicer in the docs
```suggestion
    /// (that is `a != b --> f(a) != f(b)`).
```

Reply:
Done.